github-profiles/Sorxa-9612/README.md

```diff
@@ -0,0 +1,28 @@
+# # Ciao mundo!👋
+### 💙 Acerca de mi 
+- Me interesa mucho la programación web en especial el front-end y me encanta aprender diversos temas relacionados a la tecnología. 
+- Me gustan los videojuegos.
+
+### 🎓 Educación 
+- Estudiante en ingenieria en sistemas computacionales (2022 - actualidad)
+- Aprendiz en Bootcamp  Full-stack JavaScript Developer - Guillermo Rodas en [Undefined Academy](https://undefined.academy/).
+[![Typing SVG](https://readme-typing-svg.demolab.com?font=Fira+Code&weight=200&size=19&duration=4996&pause=997&color=387EF7&center=true&vCenter=true&width=435&lines=En+proceso+de+aprendizaje...)](https://git.io/typing-svg)
+
+## Comandos
+### 5 comandos de la terminal:
+|**Comando**   | **Descripción**                                             | 
+|--------------|-------------------------------------------------------------|
+|*```ls```*    | Lista de los archivos de la carpeta actual o ruta especifica|
+| *```df```*   | Muestra el espacio libre actual del disco duro              |
+|*```cd```*    | Permite navegar entre carpetas                              |
+|*```pwd```*   | Nos muestra la ruta absoluta del directorio actual          |
+|*```mkdir```* | Crea una nueva carpeta dado un nombre                       |
+
+### Comandos alias: 
+|**Alias**   | **Descripción**                                                               |
+|------------|-------------------------------------------------------------------------------|
+| *```cl```* | alias del comando ```clear``` para limpiar la terminal                        |
+| *```rm```* | alias del comando ```trash``` evita eliminar de forma permanente los archivos |
+
+**🌐 Link perfil de github:** 
```

Comment:
Me ha gustado mucho tu creatividad, si tienes videos o tutoriales sobre esto, porfa me los haces llegar

Reply:
✨Muchas gracias, me apoye de estos tutoriales para hacer mi README: 
[CREA TU PROPIO README PARA TU PERFIL DE GITHUB + EJEMPLOS
](https://youtu.be/Rw887TSEQz0)  
[Markdown, Curso Práctico para principiantes y desarrolladores
](https://youtu.be/oxaH9CFpeEE)  
[Next Level GitHub Profile README (NEW 2022)(Markdown template) | Improve Your Github profiles](https://youtu.be/G-EGDH50hGE)